src/Perl6/Compiler.nqp

```diff
@@ -174,18 +174,18 @@ Note that only boolean single-letter options may be bundled.
 The following environment variables are respected:
 
   RAKUDOLIB   Modify the module search path
```

Comment:
We probably better remove this line until we're more certain what to do about `RAKUDOLIB`.